docs/club/resources/pr_review.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Reviewing Pull Requests
+description: Learn how to review pull requests on GitHub.
+sidebar_position: 8
+---
+
+# Reviewing Pull Requests
+
+:::note
+This guide assumes you already have your own clone of the respotiory based on your own fork (or the main project).
```

Comment:
Typo: `respotiory`

I ran a spell checker and that's the only issue as far as I can tell. Otherwise LGTM

Reply:
Should be fixed!